build/templates/grpc_session_options.rst.mako

```diff
@@ -0,0 +1,105 @@
+<%
+    import build.helper as helper
+
+    config = template_parameters['metadata'].config
+    module_name = config['module_name']
+    driver_name = config['driver_name']
+    enums = config['enums']
+%>\
+${helper.get_rst_header_snippet('gRPC Support', '=')}
+
+Support for using ${driver_name} over gRPC
+
+.. py:currentmodule:: ${module_name}
+
+
+Enums
+=====
+
+
+SessionInitializationBehavior
+-----------------------------
+
+.. py:class:: SessionInitializationBehavior
+
+    .. py:attribute:: SessionInitializationBehavior.AUTO
+
+
+
+        Attach to an existing session with the specified name if it exists, otherwise intialize a new session.
```

Comment:
I think this needs to specify that the existing or new session in on the server so there's no ambiguity for clients.
Do we have a term for sessions that live in the server?